libs/openchallenges/ui/src/lib/footer/footer.component.html

```diff
@@ -9,10 +9,18 @@
       <ul class="footer-link-group">
         <li><a href="/about">About</a></li>
         <li><a href="/team">Meet Our Team</a></li>
-        <!-- <li><a href="#">API</a></li> -->
         <li>
-          <a
+          <a 
+            href="https://openchallenges.io/api-docs"
```

Comment:
We should move `https://openchallenges.io` to the configuration file so that the app works when deployed to staging and production servers. This is tracked in #2219.

Reply:
Resolving for now, as this will addressed in another PR.